platform/iphone/os_iphone.cpp

```diff
@@ -30,11 +30,12 @@
 
 #include "os_iphone.h"
 
-#include "drivers/gles2/rasterizer_gles2.h"
+//#include "drivers/gles2/rasterizer_gles2.h"
```

Comment:
Yeah, especially when it is probably going to have a completely different interface due to the refactor of VisualServer.